docs/DreamCast_Specs.md

```diff
@@ -0,0 +1,22 @@
+# SEGA Dreamcast Specifications
+
+### CPU Specifications:
+    - **Processor**:								Hitachi SH-4 (128-bit)
```

Comment:
The Github markdown interpretation doesn't print this as a list since the dash isn't at the beginning of the line and therefore the formatting is broken.
Could you please move this (and the following entries) to the beginning of the line?

```suggestion
- **Processor**:								Hitachi SH-4 (128-bit)
```

Reply:
Sorry, it looks like I was not specific enough here.
This applies to the following list entries also. Now they are subitems of processors.
This also applies to GPU, etc. below.
Could you please change this as well?

Reply:
Yeah it was intended for the bullets to not be filled with white in their MD view. 
I changed it in any case.